stanford_profile.info.yml

```diff
@@ -44,32 +41,17 @@ install:
   - admin_toolbar_links_access_filter:admin_toolbar_links_access_filter
   - admin_toolbar_tools:admin_toolbar_tools
   - config_split:config_split
-  - profile_split_enable:profile_split_enable
   - allowed_formats:allowed_formats
   - chosen:chosen
-  - coffee:coffee
```

Comment:
I find this module helpful. Can we keep it or replace it with something similar.

Reply:
toolbar module has the same functionality, just without the keyboard shortcut

Reply:
But, but, it is the keyboard shortcut that I want.

Reply:
https://www.excelcampus.com/vba/keyboard-shortcut-run-macro/ :trollface: